### ISSUE TITLE
transform="matrix(1 0 0 -1 0 140)" throws error

### ISSUE THREAD

**Issue by maitham** [software-mansion/react-native-svg#1374]
The below code causes an error JSON value 'matrix 1 0 0 -1 0 140' of type NSString cannot be converted to a CATransform3D did you pass something other than an array?

Whats the best way of resetting the coordinate system so that it is in the bottom left corner? 

```
<Svg width={width} height={height} transform="matrix(1 0 0 -1 0 140)">
        {data.map((el, i) => {
          return (
            <>
              <Rect
                x={i * (barWidth + 10)}
                y={0}
                width={barWidth}
                height={(el.y / max) * paddedHeight}
                fill="#DFDFDF"
                rx="5"
                ry="5"
              />
              <Circle
                cx={barWidth / 2 + i * (barWidth + 10)}
                cy={(el.y2 / max) * paddedHeight}
                r="5"
                fill={colors.primaryRed}
              />
            </>
          );
        })}
      </Svg>
```

**Comment by edc123** [comment]
+1

**Comment by WoLewicki** [comment]
@maitham the types for this were misleading and they should be fixed with this commit: [`f6d5b2b` (#1914)](https://github.com/software-mansion/react-native-svg/pull/1914/commits/f6d5b2b37b6e30cff02255b1973661242f044bde). `Svg` accepts `react-native` style `transform` prop since it is not parsed by library. If you want to pass e.g. `transform="matrix(1 0 0 -1 0 140)"`, you should wrap your children of `Svg` with `G` component and pass it there since it accepts all `transform` options available in the library.

@edc123 your problem should be resolved on the newest `main`. Could you check it?

**Comment by WoLewicki** [comment]
Actually, since it is part of the svg standard to pass `transform` to `svg` element, it should be properly handled. For now I'll leave it as it is, but it might be good for the future to properly handle it.
